```
Traceback (most recent call last):
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/application.py", line 350, in build
    self.builder.build_update()
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 292, in build_update
    self.build(to_build,
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 306, in build
    updated_docnames = set(self.read())
  File "/usr/local/lib/python3.8/contextlib.py", line 120, in __exit__
    next(self.gen)
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 213, in pending_warnings
    memhandler.flushTo(logger)
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 178, in flushTo
    logger.handle(record)
  File "/usr/local/lib/python3.8/logging/__init__.py", line 1599, in handle
    self.callHandlers(record)
  File "/usr/local/lib/python3.8/logging/__init__.py", line 1661, in callHandlers
    hdlr.handle(record)
  File "/usr/local/lib/python3.8/logging/__init__.py", line 950, in handle
    rv = self.filter(record)
  File "/usr/local/lib/python3.8/logging/__init__.py", line 811, in filter
    result = f.filter(record)
  File "/tmpfs/src/github/python-bigquery/.nox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 423, in filter
    raise exc
sphinx.errors.SphinxWarning: /tmpfs/src/github/python-bigquery/docs/usage/encryption.rst:13:start-after pattern not found: [START bigquery_create_table_cmek]

Warning, treated as error:
/tmpfs/src/github/python-bigquery/docs/usage/encryption.rst:13:start-after pattern not found: [START bigquery_create_table_cmek]
nox > Command sphinx-build -W -T -N -b html -d [docs/_build/doctrees/](http://docs/_build/doctrees/) [docs/](http://docs/) [docs/_build/html/](http://docs/_build/html/) failed with exit code 2
nox > Session docs failed.
```

--- a/docs/usage/encryption.rst
+++ b/docs/usage/encryption.rst
@@ -10,7 +10,7 @@ in the BigQuery documentation for more details.
 Create a new table, using a customer-managed encryption key from
 Cloud KMS to encrypt it.
 
-.. literalinclude:: ../snippets.py
+.. literalinclude:: ../samples/snippets/create_table_cmek.py
    :language: python
    :dedent: 4
    :start-after: [START bigquery_create_table_cmek]

